openetl_utils/deprecated/form_utils.py

```
@@ -10,8 +10,8 @@
 """
 import streamlit as st
 
-from utils.deprecated.api_utils import send_request
-from utils.enums import *
+from openetl_utils.deprecated.api_utils import send_request
+from openetl_utils.enums import *
```

Ruff (0.8.2)

`from openetl_utils.enums import *` used; unable to detect undefined names

(F403)